bokehjs/src/coffee/common/plot.coffee

```diff
@@ -62,6 +74,11 @@ class PlotView extends ContinuumView
 
     @canvas_view.render()
 
+    # If requested, try enabling webgl
```

Comment:
It would be good to avoid comments like this one. The code below is pretty much self-explanatory.